lib/registry.js

```diff
@@ -27,10 +28,12 @@ class Registry {
 		const type = ['#', 'TYPE', name, item.type].join(' ');
 
 		const values = (item.values || []).reduce((valAcc, val) => {
-			const labels = Object.keys(val.labels || {}).map(
+			let labels = Object.keys(val.labels || {}).map(
 				key => `${key}="${escapeLabelValue(val.labels[key])}"`
 			);
 
+			labels = Object.assign(this._defaultLabels, labels);
```

Comment:
not sure `Object.assign` makes sense here, they are arrays not objects. should be `this._defaultLables.concat(labels)`. Unsure how to handle dupes, though? I can see from the tests that it works, but it feels weird to me